routes/api/v2/authentication.coffee

```diff
@@ -9,7 +9,11 @@ Meteor.startup ->
           user = Meteor.users.findOne(apiKey.owner)
           # @authenticatedUSer = user
           @next()
-        else lib.endWithError @response, 401, "Not authenticated"
+        else if(key in Meteor.call "getDeployKeys") 
```

Comment:
No need to Meteor.call when on the server. Just call the function getDeployKeys directly.